src/types/NodeType.js

```diff
@@ -233,8 +240,15 @@ export default function createNodeType({
       resourceConfig,
       ...config
     }: NodeTypeConfig<R>) {
-      // eslint-disable-next-line no-param-reassign
-      localIdFieldName = getLocalIdFieldName(name, localIdFieldName);
+      if (localIdFieldMode !== 'omit') {
+        // eslint-disable-next-line no-param-reassign
+        localIdFieldName = getLocalIdFieldName(name, localIdFieldName);
+      } else {
+        invariant(
+          !localIdFieldName,
```

Comment:
this is fine because `makeId` actually is the thing that produces the local id for the global id resolver?

Reply:
yup... these are just schemas so none of the fields interact (or can interact)